src/WebJobs.Extensions.DurableTask/Listener/DurableTaskScaleMonitor.cs

```diff
@@ -81,7 +87,7 @@ public async Task<DurableTaskTriggerMetrics> GetMetricsAsync()
                 DisconnectedPerformanceMonitor performanceMonitor = this.GetPerformanceMonitor();
                 heartbeat = await performanceMonitor.PulseAsync();
             }
-            catch (StorageException e)
+            catch (Exception e) when (e.InnerException != null && e.InnerException.GetType() == typeof(RequestFailedException))

```

Comment:
I think in the new version of DTFx I've standardized all of the exceptions thrown by the underlying storage clients to throw `DurableTaskStorageException`. I am unsure if this is the best way to catch it haha

Reply:
Yes -- I am unsure why I made this so verbose on the first pass lol